src/main/java/org/springframework/samples/petclinic/model/Game.java

```diff
@@ -0,0 +1,91 @@
+package org.springframework.samples.petclinic.model;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+import org.hibernate.annotations.ColumnDefault;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.samples.petclinic.model.enums.GameState;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "games")
+public class Game {
+
+    @Id
+	@GeneratedValue(strategy = GenerationType.UUID)
+    protected String id; 
+
+    @Size(min = 3, max = 50)
+	@NotBlank
+	@Column(name = "name")
+	private String name;
+
+    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    LocalDateTime start;
+
+    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
```

Comment:
```suggestion
    @DateTimeFormat(pattern = "dd-MM-yyyy HH:mm:ss")
    LocalDateTime start;

    @DateTimeFormat(pattern = "dd-MM-yyyy HH:mm:ss")
```

No usamos el de España por alguna razón?

Reply:
Por ninguna la verdad jajajaj